src/main/resources/report.csv

```diff
@@ -0,0 +1,3 @@
+fruit,amount
+banana,70
+apple,30
```

Comment:
lets move everything related to strategy in strategy package and create there impl package for operation implementations